TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift

```diff
@@ -107,7 +107,7 @@ extension AuthenticatedSecureStorage: AuthenticatedSecureStorageProvider {
         }
 
         Logger.shared.log(withLevel: .error, message: error.localizedDescription)
-        if let biometricError = BiometricError.given(error as NSError) {
+        if let keychainError = error as? KeychainError, let biometricError = BiometricError.given(keychainError){
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/389d70a047903254b33925b4a9b06b5a91b8aa1c/TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift#L110"></span>Opening braces should be preceded by a single space and on the same line as the declaration.<br />
<code>opening_brace</code> <code>AuthenticatedSecureStorage.swift:110</code></td>
    </tr>
  </tbody>
</table>